pynest/examples/pong/run_simulations.py

```diff
@@ -0,0 +1,252 @@
+# -*- coding: utf-8 -*-
+#
+# run_simulations.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+r"""Application training networks to play pong against each other.
+----------------------------------------------------------------
+This program makes two spiking neural networks of two layers each compete
+against each other in encoding an input-output mapping within their weights.
+
+The script is centered around a simulation of the classic game Pong
+in which the vertical position of the ball determines the input for both
+'players' and their output activation after a predetermined simulation
+time changes the paddle positions within the game.
+
+The output of the script is stored in three .pkl files which contain information
+about the state of the game, the left network and the right network respectively
+after every simulation step which can be used to visualize the output
+(e.g. using :doc:`generate_gif.py <./generate_gif.py>`).
+
+The idea for this simulation, as well as the core of the R-STDP and Pong
+implementation are from [1]_ and were created by Timo Wunderlich and Electronic
+Vision(s) (The original implementation can be found
+'here <https://github.com/electronicvisions/model-sw-pong>'_).
```

Comment:
```suggestion
`here <https://github.com/electronicvisions/model-sw-pong>`_).
```

Note: For links and internal references you have to use the backticks not apostrophe